### ISSUE TITLE
circleci: failed tests (14128): TestClusterRecovery TestGossipPeerings 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#5021]
The following test appears to have failed:

[#14128](https://circleci.com/gh/cockroachdb/cockroach/14128):

```
I160309 18:12:05.653789 acceptance/chaos_test.go:178  client 1 shutting down
I160309 18:12:05.653888 acceptance/chaos_test.go:178  client 2 shutting down
I160309 18:12:05.655643 acceptance/chaos_test.go:178  client 0 shutting down
I160309 18:12:05.679572 acceptance/chaos_test.go:361  1469 292.2/sec
I160309 18:12:06.679847 acceptance/cluster/localcluster.go:585  stopping
--- FAIL: TestClusterRecovery (11.63s)
	localcluster.go:557: expected event &{2 die}, got <nil> (after [])
	localcluster.go:557: expected event &{2 restart}, got <nil> (after [])
=== RUN   TestNodeRestart
I160309 18:12:08.227705 acceptance/cluster/localcluster.go:254  Initializing Cluster:
Cluster Setup: AdHoc 3x1
Duration: 5s, Stall: 2m0s
Node 0 - Store 0
Node 1 - Store 0
Node 2 - Store 0
Total Nodes Count:3, Total Store Count:3
--
I160309 18:12:24.837782 acceptance/gossip_peerings_test.go:124  restarting node 0
I160309 18:12:27.224358 acceptance/gossip_peerings_test.go:135  restarting node 1
I160309 18:12:29.550839 acceptance/gossip_peerings_test.go:124  restarting node 0
I160309 18:12:31.904473 acceptance/gossip_peerings_test.go:135  restarting node 2
I160309 18:12:34.378038 acceptance/cluster/localcluster.go:585  stopping
--- FAIL: TestGossipPeerings (15.49s)
	localcluster.go:557: expected event &{0 die}, got <nil> (after [])
=== RUN   TestGossipRestart
--- SKIP: TestGossipRestart (0.00s)
	gossip_peerings_test.go:147: TODO(pmattis): #4445
=== RUN   TestDockerJava
I160309 18:12:35.920468 acceptance/cluster/localcluster.go:254  Initializing Cluster:
Cluster Setup: AdHoc 3x1
Duration: 5s, Stall: 2m0s
Node 0 - Store 0
Node 1 - Store 0
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:64: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

**Comment by tamird** [comment]
cc @vivekmenezes for TestClusterRecovery

**Comment by petermattis** [comment]
Fixed by #5041.
